### DISCUSSION TITLE
ViewHandler - InvalidOperationException : Context cannot be null here

### DISCUSSION THREAD

**Discussion by JaehwanMango** [dotnet/maui#18744] [Q&A]
#### `When does the exception happen` : _navigating to page with customized controls_
`Platform` : _Android_
`SDK` : _8.x Preview Version_
`Occurred Location` :  [_ViewHandlerOfT.Android - Line12_](https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/View/ViewHandlerOfT.Android.cs#L12)

```cs
// ViewHandlerOfT.Android - Line12
public partial class ViewHandler<TVirtualView, TPlatformView> : IPlatformViewHandler
{
        // Line 12
	public Context Context => MauiContext?.Context ?? throw new InvalidOperationException($"Context cannot be null here");
        ...
}
```

## Question

* If you don't understand, please `let me know what you need`.
* There is `no Handler registered` to customized control. `Does it need handler`?
* How to `(Context)null check?`
* Please let me know `when the context can be null`

**Comment by SSstupid** [comment]
`Sample link` : https://github.com/SSstupid/IDontKnow-This-/pull/5
